DataFormats/Scouting/src/classes_def.xml

```diff
@@ -54,6 +55,7 @@
   <class name="Run3ScoutingTrack" ClassVersion="3">
       <version ClassVersion="3" checksum="3352318277"/>
   </class>
+  <class name="Run3ScoutingHitPatternPOD"/>
```

Comment:
This doesn't need versioning because it's a struct?

Reply:
Without this addition, one would get a failure as no dictionary would exist for the corresponding `Run3ScoutingMuon` data member. Hence, I'm leaving it there

Reply:
I was asking about the fact that it does not have `"ClassVersion="X"` and checksum (not suggesting to remove).

Reply:
Apologies, I misread. Done.